examples/extension/build.gradle

```diff
@@ -30,6 +30,7 @@ ext {
 }
 
 repositories {
+  mavenLocal()
```

Comment:
Remove or order at bottom?

Reply:
Will remove in later PR when we have suitable snapshots published.